test/Swashbuckle.AspNetCore.Cli.Test/ToolTests.cs

```diff
@@ -9,6 +9,20 @@ namespace Swashbuckle.AspNetCore.Cli.Test
 {
     public static class ToolTests
     {
+        [Fact]
+        public static void Can_Output_Swagger_Document_Names()
+        {
+            var result = RunListCommand((outputPath) =>
+            [
+                "list",
+                "--output",
+                outputPath,
+                Path.Combine(Directory.GetCurrentDirectory(), "MultipleVersions.dll")
+            ], nameof(Can_Output_Swagger_Document_Names));
+            var expected = $"Swagger Document Names:{Environment.NewLine}\"1.0\"{Environment.NewLine}\"2.0\"{Environment.NewLine}";
```

Comment:
I wonder if it's better to not have the `Swagger Document Names:` header so people could easily pipe the output into other tools?

Reply:
I was thinking of spitting out content similar to CSV output, but I'm not hung up on the exact output shape too much one way or the other. So, I'll remove the header line.